Traceback (most recent call last):
  File "/home/jovyan/jonglae/Retry1/Segmentation/33333.py", line 213, in <module>
    main()
  File "/home/jovyan/jonglae/Retry1/Segmentation/33333.py", line 198, in main
    train_sam_with_image_mask(
  File "/home/jovyan/jonglae/Retry1/Segmentation/33333.py", line 121, in train_sam_with_image_mask
    for images, measurements in dataloader:
  File "/home/jovyan/.conda/envs/aipbase/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 701, in __next__
    data = self._next_data()
  File "/home/jovyan/.conda/envs/aipbase/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 757, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/jovyan/.conda/envs/aipbase/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 52, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/jovyan/.conda/envs/aipbase/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 52, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/jovyan/jonglae/Retry1/Segmentation/33333.py", line 87, in __getitem__
    measurement = load_measurements(self.measure_xml_paths[idx], original_shape, resize_shape=self.resize_shape)
  File "/home/jovyan/jonglae/Retry1/Segmentation/33333.py", line 46, in load_measurements
    tree = ET.parse(measure_xml_path)
  File "/home/jovyan/.conda/envs/aipbase/lib/python3.10/xml/etree/ElementTree.py", line 1222, in parse
    tree.parse(source, parser)
  File "/home/jovyan/.conda/envs/aipbase/lib/python3.10/xml/etree/ElementTree.py", line 569, in parse
    source = open(source, "rb")
FileNotFoundError: [Errno 2] No such file or directory: '/home/jovyan/data-vol-1/Y_T_Result/SAM_Train/measure/image95_overlay.xml'
